sw/device/tests/BUILD

```diff
@@ -626,6 +626,34 @@ opentitan_functest(
     ],
 )
 
+opentitan_functest(
+    name = "csrng_edn_concurrency_test",
+    srcs = ["csrng_edn_concurrency_test.c"],
+    cw310 = cw310_params(timeout = "long"),
```

Comment:
Just a heads up, because this a "long" test, it [will not run in CI](https://cs.opensource.google/opentitan/opentitan/+/master:ci/scripts/run-fpga-cw310-tests.sh;drc=46b156a66e1d0e4f7e651b55790347a8fae4fbc0;l=42). If there is a way to bring the run time down then it will get automatically run. If not, we could make an exception and manually run this test in CI if that is desired.

Reply:
Updated test parameters to be ale to run this test in CI.